upnotify-bot/src/main/java/utils/WebUtils.java

```diff
@@ -127,6 +130,24 @@ public String getStringFromUrlAndSelectorPathJsoup(String url, String selectorPa
 		Element el = getElementFromUrlAndSelectorPathJsoup(url, selectorPath);
 		return el.text();
 	}
+
+	public String getStringFromUrlAndSelectorPathUsingJsoupAndSelenium(String url, String selectorPath) {
+		//URL driverURL = getClass().getClassLoader().getResource("chromedriver.exe");
+		//System.out.println("URL:" + driverURL);
+		System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver.exe");
```

Comment:
Try to use something like "ClassLoader.getSystemResourceAsStream" instead of typing the whole path "src/main/resources/chromedriver.exe" yourself.
e.g.

public UpnotifyBot() {
		System.out.println("Constructing the bot...");
		InputStream ins = ClassLoader.getSystemResourceAsStream("SECRET_KEYS/Keys.properties");
		Properties prop = new Properties();
		try {
			prop.load(ins);
...